cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ResponseData.java

```diff
@@ -1,21 +1,22 @@
-package uk.gov.hmcts.cmc.claimstore.models;
+package uk.gov.hmcts.cmc.domain.models;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonUnwrapped;
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 import org.hibernate.validator.constraints.NotBlank;
-import uk.gov.hmcts.cmc.claimstore.models.legalrep.StatementOfTruth;
-import uk.gov.hmcts.cmc.claimstore.models.party.Party;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.utils.ToStringStyle;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.gov.hmcts.cmc.domain.utils.ToStringStyle'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)